pallets/tendermint-client/src/types.rs

```diff
@@ -88,7 +88,7 @@ impl Decode for TMClientStorageWrapper {
     fn decode<I: Input>(input: &mut I) -> Result<Self, Error> {
         let len = input.remaining_len().unwrap().ok_or_else(|| "meh")?;
         let mut vec: Vec<u8> = Vec::with_capacity(len);
-        vec.resize_default(len);
+        vec.resize_with(len, Default::default);
```

Comment:
@ParthDesai this feature seems to be gone from nightly, but the equivalent behavior seems to be provided by `resize_with`